docs/examples/applications/values_dialog.py

```diff
@@ -0,0 +1,21 @@
+"""# Input values dialog.
+
+A basic example of a user input dialog.
+
+This will pause code execution until the user responds.
+
+# ![Values input dialog](../../images/values_input.png){ width=50% }
+
+```python linenums="1"
+from magicgui.widgets import request_values
+
+vals = request_values(
+    age=int,
+    name={"annotation": str, "label": "Enter your name:"},
+    title="Hi, who are you?",
+)
+print(repr(vals))
+```
+"""
+
+# %%
```

Comment:
Here is an example of an almost, but not quite entirely useless `# %%`
If it is missing, mkdocs-gallery freaks out and puts a big "BROKEN" image in the examples gallery. If we have the `# %%` line here, it is treated like an empty code cell with no output, which it is. Then you get the default not-scary image next to this example in the gallery page.